### PR TITLE
added the documentation of the get_existing_objects

### PR DESCRIPTION
**Pull request by lemonade-666**
added the documentation of the get_existing_objects

### CODE REVIEW

File: S3/s3upload.py [hunk 1]
```diff
@@ -14,6 +14,14 @@
 # TODO tidy up the entire code and reinvent the function documentions
 # this function return the list of objects (files) already existing in the given bucket
 def get_existing_objects(bucket_name):
+    """[this is]
```
Comment (lemonade-666) [inline S3/s3upload.py:17]: testing

Comment (nrookwood) [reply S3/s3upload.py:17]: testing_testing

**Review by nrookwood — CHANGES_REQUESTED** [review]
please see the test

File: S3/s3upload.py (hunk 1 above)
Comment (lemonade-666) [reply S3/s3upload.py:17]: this is done

**Review by lemonade-666 — COMMENTED** [review]
changed

File: S3/s3upload.py [hunk 2]
```diff
@@ -14,6 +14,14 @@
...
 # this function return the list of objects (files) already existing in the given bucket
 def get_existing_objects(bucket_name):
+    """[this is]
+
+    Args:
+        bucket_name ([type]): [description]
+
+    Returns:
+        [type]: [description]
+    """
```
Comment (nrookwood) [inline S3/s3upload.py:24]: Sweet nice test

**Review by nrookwood — APPROVED** [review]

**Review by nrookwood — APPROVED** [review]

**Pull request merged**